src/full/Agda/Compiler/Common.hs

```diff
@@ -4,7 +4,7 @@ module Agda.Compiler.Common where
 
 import Prelude hiding ((!!))
 
-import Data.List as List hiding ((!!))
+import Data.List as List (sortBy, isPrefixOf)
```

Comment:
Good, this is in line with`-Wcompat`!  Maybe the `as List` isn't even needed anymore?

Reply:
Had to fix it otherwise `cabal v2-repl` wouldn't even start :slightly_smiling_face: